### DISCUSSION TITLE
Rating component - select 0 stars for non controlled usage

### DISCUSSION THREAD

**Discussion by HananoshikaYomaru** [mantinedev/mantine#2862] [Q&A] [answered]
How to select 0 stars?

**Comment by rtivital** [comment, marked as answer]
You can use controlled component and set any values that you need – https://mantine.dev/core/rating/#controlled

**Reply by HananoshikaYomaru** [reply]
But it seems like 0 stars can never be selected even for the controlled usage.

**Reply by rtivital** [reply]
In controlled component you fully control value on your side, set it to zero with `setValue(0)` at any time

**Reply by HananoshikaYomaru** [reply]
OK. Then I interpret this as selecting 0 is not achievable in non controlled usage.

**Reply by HananoshikaYomaru** [reply]
@rtivital my question is not well written. Sorry about that.
